src/web_app_skeleton/src/actions/errors/show.cr.ecr

```diff
@@ -0,0 +1,52 @@
+class Errors::Show < Lucky::ErrorAction
+  def handle_error(error : JSON::ParseException)
+    message = "There was a problem parsing the JSON." \
+            " Please check that it is formed correctly"
+
+    <%- if @api_only -%>
+    json Errors::ShowSerializer.new(message), status: 400
+    <%- else -%>
+    if json?
+      json Errors::ShowSerializer.new(message), status: 400
+    else
+      render_error_response status: 500
+    end
+    <%- end -%>
+  end
+
+  def handle_error(error : Lucky::RouteNotFoundError)
+    <%- if @api_only -%>
+    json Errors::ShowSerializer.new("Not found"), status: 404
+    <%- else -%>
+    if json?
+      json Errors::ShowSerializer.new("Not found"), status: 404
+    else
+      render_error_response title: "Sorry, we couldn't find that page.", status: 404
+    end
+    <%- end -%>
+  end
+
+  def handle_error(error : Exception)
+    error.inspect_with_backtrace(STDERR)
+
+    message = "An unexpected error occurred"
+
+    <%- if @api_only -%>
+    json Errors::ShowSerializer.new(message), status: 500
+    <%- else -%>
+    if json?
+      json Errors::ShowSerializer.new(message), status: 500
+    else
+      render_error_response status: 500
+    end
+    <%- end -%>
+  end
+
+  private def render_error_response(status : Int32, title : String = "We're sorry. Something went wrong.")
+    <%-if @api_only -%>
```

Comment:
I think if it is api only you can remove this method completely since it is no longer used